### DISCUSSION TITLE
🖊️ 'op willekeurig'

### DISCUSSION THREAD

**Discussion by MarleenGilsing** [hedyorg/hedy#5999] [Language Ideas]
At the moment at random translates as op willekeurig, which seems weird. 
Maybe just willekeurig?

**Comment by Felienne** [comment]
Hi Marleen!

I see what you mean (although in English it does translate to the same)

For technical reasons, currently we do need some word to indicate the position, so it needs to be two words. We could do something like "op willekeurige plek" but that is a bit long?

**Comment by MarleenGilsing** [comment]
What about 'kies willekeurig'?
